src/Uno.UI/Microsoft/UI/Xaml/Controls/NavigationView/NavigationView.cs

```diff
@@ -1168,6 +1168,21 @@ internal void OnNavigationViewItemInvoked(NavigationViewItem nvi)
 
 			if (updateSelection)
 			{
+				var indicatorTarget = nvi;
+
+				// Move indicator to topmost collapsed parent
+				var parent = GetParentNavigationViewItemForContainer(nvi);
```

Comment:
Is this coming from WinUI's original source ?

Reply:
Not **yet** 👍  . Will be backporting this fix to MUX repo